docs/data/rpc/admin-guide.mdx

```diff
@@ -305,15 +305,15 @@ docker run -p 8001:8001 -p 8000:8000 \
 
 #### Configuration
 
-For production, we recommend running Soroban RPC with a [TOML](https://toml.io/en/) configuration file rather than CLI flags. This is similar to creating a configuration file for Stellar-Core as we did previously. For example, using [our docker image](https://hub.docker.com/r/stellar/soroban-rpc):
+For production, we recommend running Stellar RPC with a [TOML](https://toml.io/en/) configuration file rather than CLI flags. This is similar to creating a configuration file for Stellar-Core as we did previously. For example, using [our docker image](https://hub.docker.com/r/stellar/soroban-rpc):
 
 ```bash
 docker run -p 8001:8001 -p 8000:8000 \
 -v <CONFIG_FOLDER>:/config stellar/soroban-rpc \
 --config-path <RPC_CONFIG_FILENAME>
 ```
 
-You can use Soroban RPC itself to generate a starter configuration file:
+You can use Stellar RPC itself to generate a starter configuration file:
 
 ```bash
 docker run stellar/soroban-rpc gen-config-file > soroban-rpc-config.toml
```

Comment:
Same for here. Might want to change "stellar/soroban-rpc" to "stellar/stellar-rpc" and the .toml file naming too